src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+type PrepareTodo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number
+  user?: {
```

Comment:
If the `user` is not there, then the user will not be able to add a todo, so "?" there is no sense in the user property. A user will always exist if a todo is created.